### ISSUE TITLE
SELECT Query Without a Table Produces No Results

### ISSUE THREAD

**Issue by apavlo** [cmu-db/terrier#713]
We are not able to execute SELECT queries correctly if we do not provide a FROM clause. These types of queries are important and are executed often by ORMs and other app frameworks.

```
terrier=# SELECT 1;
 INTEGER 
---------
(0 rows)
```

```
terrier=# SELECT NULL;
ERROR:  binding failed
```

**Comment by 17zhangw** [comment]
For `SELECT 1`, this is due to the way how `TableFreeScan` does not have a plan. In fact with projections being fixed in #705, `SELECT 1` will actually cause a segfault.

**Comment by apavlo** [comment]
`SELECT 1` is working. `SELECT NULL` fails. I will submit a new bug.
